packages/react/src/runtimes/remote-thread-list/cloud/CloudThreadListItemRuntime.tsx

```
@@ -0,0 +1,89 @@
+    private adapter: CloudThreadListItemRuntimeAdapter,
+    private threadListItemRuntime: ThreadListItemRuntime,
+  ) {}
+
+  public __internal_setThreadListItemRuntime(
+    threadListItemRuntime: ThreadListItemRuntime,
+  ) {
+    this.threadListItemRuntime = threadListItemRuntime;
+  }
+
+  public async getOrCreateThread() {
+    let threadData = this.threadListItemRuntime.getState();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'threadData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)